src/WorkOS.net/Services/SSO/Entities/Connection.cs

```diff
@@ -19,11 +20,18 @@ public class Connection
         [JsonProperty("id")]
         public string Id { get; set; }
 
+        /// <summary>
+        /// The linked state of the Connection.
+        /// </summary>
+        [JsonProperty("state")]
+        public ConnectionState State { get; set; }
+
         /// <summary>
         /// The linked status of the Connection.
         /// </summary>
         [JsonProperty("status")]
-        public ConnectionStatus Status { get; set; }
+        [ObsoleteAttribute("The status property is obsolete. Please use state instead.", false)]
+        public ConnectionState Status { get; set; }
```

Comment:
The enum value here should be `ConnectionStatus`:

```suggestion
        public ConnectionStatus Status { get; set; }
```